### PR TITLE
Hgc tpg integration 190326 sw

### PR DESCRIPTION
**Pull request by snwebb**
Hi Jean-Baptiste,

I have made the changes as suggested by Kevin, other than
https://github.com/cms-sw/cmssw/pull/26266#discussion_r270870768

~For some reason there are still many other commits in the branch although I followed your instructions. Are you able to just take the relevant changes from here (i.e. from L1Trigger/L1THGCal)?~

### CODE REVIEW

**Comment by snwebb** [comment]
Sorry I had the wrong base, everything is fine now

File: L1Trigger/L1THGCal/src/backend/HGCalHistoClusteringImpl.cc [hunk 1]
```diff
@@ -43,18 +43,19 @@ float HGCalHistoClusteringImpl::dR( const l1t::HGCalCluster & clu,
...
 
-    for(auto & clu : clustersPtrs){
+    for(const auto& clu : clustersPtrs){
 
         int z_side = triggerTools_.zside(clu->detId());
 
 
-        double radiusCoefficientA = dr_byLayer_coefficientA_.empty() ? dr_ : dr_byLayer_coefficientA_.at(triggerTools_.layerWithOffset(clu->detId())); // use at() to get the assert, for the moment
-        double radiusCoefficientB = dr_byLayer_coefficientB_.empty() ? 0 : dr_byLayer_coefficientB_.at(triggerTools_.layerWithOffset(clu->detId())); // use at() to get the assert, for the moment
+        double radiusCoefficientA = dr_byLayer_coefficientA_.empty() ? dr_ : dr_byLayer_coefficientA_.at[triggerTools_.layerWithOffset(clu->detId())];
```
Comment (jbsauvan) [inline L1Trigger/L1THGCal/src/backend/HGCalHistoClusteringImpl.cc]: It doesn't compile. `at()` should be replaced by `[]`

Comment (snwebb) [reply L1Trigger/L1THGCal/src/backend/HGCalHistoClusteringImpl.cc]: right, sorry, fixed.

**Comment by jbsauvan** [comment]
Thanks. Could you now squash the commits? Then I will cherry-pick, add my commit, and push directly to the integration branch.

**Comment by snwebb** [comment]
Done

**Pull request closed without merging**